client/wp-admin-scripts/beta-features-tracking-modal/container.js

```diff
@@ -0,0 +1,109 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { useState, useEffect, useRef } from '@wordpress/element';
+import { Button, Modal, CheckboxControl } from '@wordpress/components';
+import { withDispatch } from '@wordpress/data';
+import { compose } from '@wordpress/compose';
+import { OPTIONS_STORE_NAME } from '@woocommerce/data';
+
+const BetaFeaturesTrackingModal = ( { updateOptions } ) => {
+	const [ isModalOpen, setIsModalOpen ] = useState( false );
+	const [ isChecked, setIsChecked ] = useState( false );
+	const enableNavigationCheckbox = useRef(
```

Comment:
Saw your comment about this in the PR description and agree that it might be useful to target all checkboxes in this section, but this can wait for a future PR. 👍